{{cookiecutter.project_slug}}/backend/app/api/api_v1/routers/tests/test_auth.py

```diff
@@ -14,6 +14,21 @@ def verify_password_mock(first: str, second: str):
     assert response.status_code == 200
 
 
+def test_signup(client):
+    response = client.post(
+        "/api/signup",
+        data={"username": 'some@email.com', "password": "randompassword"},
+    )
+    assert response.status_code == 200
+
+    # Try again with same credentials
+    response = client.post(
```

Comment:
The already exists test can also use the `test_user` fixture and be a separate test and I don't think it should add much time.